### PR TITLE
Resolves #3144 - Add augmentation support to build_sdb.py

### LINKED ISSUE #3144: Add augmentation support to build_sdb.py
`build_sdb.py` should also support sample-level augmentations. This would allow creation of synthetic test sets.

### PR DESCRIPTION
**Pull request by tilmankamp**
X-DeepSpeech: NOBUILD

### CODE REVIEW

File: bin/build_sdb.py [hunk 1]
```diff
@@ -17,17 +18,28 @@
...
+)
 
 AUDIO_TYPE_LOOKUP = {"wav": AUDIO_TYPE_WAV, "opus": AUDIO_TYPE_OPUS}
 
 
 def build_sdb():
     audio_type = AUDIO_TYPE_LOOKUP[CLI_ARGS.audio_type]
+    augmentations = parse_augmentations(CLI_ARGS.augment)
+    if any(not isinstance(a, SampleAugmentation) for a in augmentations):
+        print("Warning: Some of the augmentations cannot be applied by this command.")
```
Comment (reuben) [inline bin/build_sdb.py:34]: Why? Which ones? Include some hint for how to fix this problem.

File: bin/build_sdb.py [hunk 2]
```diff
@@ -67,6 +79,11 @@ def handle_args():
         help="If to build an SDB with unlabeled (audio only) samples - "
         "typically used for building noise augmentation corpora",
     )
+    parser.add_argument(
+        "--augment",
+        action='append',
```
Comment (reuben) [inline bin/build_sdb.py:84]: nit: mixing double and single quotes in this file

Comment (tilmankamp) [reply bin/build_sdb.py:84]: Will fix it soon with the next PR on this file... :-)

File: bin/build_sdb.py (hunk 1 above)
Comment (tilmankamp) [reply bin/build_sdb.py:34]: Will fix it soon with the next PR on this file...

**Pull request merged**